src/Support/Arr/Order.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * This file is part of the Phalcon.
+ *
+ * (c) Phalcon Team <team@phalcon.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Phalcon\Support\Arr;
+
+use function array_values;
+use function is_object;
+use function krsort;
+use function ksort;
+
+/**
+ * Class Order
+ *
+ * @package Phalcon\Support\Arr
+ */
+class Order
+{
+    /**
+     * Sorts a collection of arrays or objects by key
+     *
+     * @param array  $collection
+     * @param mixed  $attribute
+     * @param string $order
+     *
+     * @return array
+     */
+    public function __invoke(
+        array $collection,
+        $attribute,
+        string $order = 'asc'
```

Comment:
Should we add sort flags as well? https://www.php.net/manual/en/function.sort.php#refsect1-function.sort-parameters
How about constants for the asc/desc?

Reply:
Added sort flags in the method - same as the ones from PHP